app/assets/v2/css/buttons.css

```diff
@@ -1,13 +1,13 @@
 /*
-	https://developer.mozilla.org/en/docs/Web/CSS/box-shadow
```

Comment:
@developerfred can you not include any auto-formatting changes from your IDE? thanks!

Reply:
sure!